bitnami/postgresql/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 11.0.4
+version: 11.0.5
```

Comment:
Given that this is implementing a new feature, we should at least increase the minor version.

Reply:
I don't think it's a new feature but removing a blocking condition that made no sense.